bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml

```
@@ -84,8 +84,94 @@ metadata:
                 "name": ""
               },
               "serviceMonitor": {
-                "enabled": true
-              }
+                "enabled": true,
+                "metricRelabelings": [
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_GPU_UTIL",
+                    "replacement": "accelerator_gpu_utilization",
+                    "sourceLabels": [
+                      {
+                        "__name__",
+                      }
+                    ],
```

The syntax is incorrect here. `sourceLabels` is a list of strings. Please update throughout.
```suggestion
                    "sourceLabels": [
                        "__name__"
                    ],
```

---

fixed